### PR TITLE
Errata v1.1 update

### PR DESCRIPTION
**Pull request by woodbe**
This set of changes is specific to the errata 1.1 update. The changes all seemed to revolve around FCS_RBG as far as the changes that apply to the DSC are concerned.

### CODE REVIEW

**Comment by jvdsn** [comment]
Errata v1.1 of the Crypto Catalog?

**Comment by woodbe** [comment]
Of the CC:2022.

https://www.commoncriteriaportal.org/files/ccfiles/CCMB-2024-002-v1_1-Errata_Interpretation_CC_CEM_2022.pdf

Basically we need to incorporate these changes from this update (published in June, I guess). There were not a lot of additional changes, and for DSC, it was the FCS_RBG.

File: cPP_DSC.adoc [hunk 1]
```diff
@@ -1177,7 +1177,7 @@ FCS_COP.1.1/SKC:: The TSF shall perform [_symmetric-key encryption/decryption_]
 
 FCS_RBG.1 Random Bit Generation (RBG)
 
-FCS_RBG.1.1:: The TSF shall perform deterministic random bit generation services using [*selection*: _RBG algorithm_] in accordance with [*selection*: _list of standards_] after initialization with a seed.
+FCS_RBG.1.1:: The TSF shall perform deterministic random bit generation services using [*selection*: _DRBG algorithm_] in accordance with [*selection*: _list of standards_] after initialization *with a seed*.
```
Comment (jvdsn) [inline cPP_DSC.adoc:1180]: the addition of "with a seed" is a refinement on the CC22 SFR?

**Review by jvdsn — APPROVED** [review]

File: cPP_DSC.adoc (hunk 1 above)
Comment (woodbe) [reply cPP_DSC.adoc:1180]: Yes it is

**Pull request merged**